.gitattributes

```diff
@@ -0,0 +1,2 @@
+# format on staging
+*.js	filter=deno-fmt
```

Comment:
Does this file need to be copied to the user's project?

Reply:
The filter won't be used without it.